server/service/src/stock_line/update.rs

```diff
@@ -168,6 +193,45 @@ fn generate_location_movement(
     Ok(movement)
 }
 
+fn generate_barcode_row(
+    connection: &StorageConnection,
+    existing: StockLineRow,
+    value: String,
```

Comment:
Btw, we had an issue awhile ago, for general way to allow optional nullables, but never got around to it